design.md

```diff
@@ -0,0 +1,38 @@
+# Epiverse visual map
+
+## User persona & user story
+
+This work is mostly targeted at the following persona:
+
+- PEI tool implementer
+- PEI tool creator
+- domain expert
+
+with the following user story:
+
+> I want to quickly & visually understand what Epiverse is about, and its niche in the ecosystem of data science tools for Epidemiology
+
+## Design options and user feedback
+
+3 options have been identified to address the user stories:
+
+- cards
+- network
+- map
```

Comment:
Could we be slightly more precise about what these terms mean? 

I can map them onto what I know, but I want to ensure we do not have any discrepancies that pop up down the line.

I make an attempt to put these terms into our specific context below: 

```suggestion
- cards - element containing content and actions about one data science tool for Epidemiology
- network - a compilation of nodes and edges, containing all data science tools for Epidemiology and how they are dependent on each other
- map - ???
```

Reply:
Solved by [`9d18e5b` (#4)](https://github.com/epiverse-connect/epiverse-map/pull/4/commits/9d18e5b72a6d53db0f6064da0be47b4bb789e33f).